### PR TITLE
[Test automation] Verify Azure Container Registry plugin is supported

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Description

[Test automation] Verify Azure Container Registry plugin is supported

## Which issue(s) does this PR fix
https://issues.redhat.com/browse/RHIDP-771

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: e2e-tests/playwright/e2e/plugins/acr.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    await uiHelper.openSidebar('Catalog');
+    await uiHelper.verifyHeading('My Org Catalog');
+    await uiHelper.selectMuiBox('Kind', 'component');
+    await uiHelper.clickLink('acr-test-entity');
+    await uiHelper.clickTab('Image Registry');
+    await uiHelper.verifyHeading(
+      'Azure Container Registry Repository: hello-world',
+    );
+    await uiHelper.verifyRowInTableByUniqueText('latest', [
+      'Jan 18, 2024, 8:26 PM',
```
Comment (gustavolira) [inline e2e-tests/playwright/e2e/plugins/acr.spec.ts:30]: I believe setting this date here makes our test quite fragile and prone to breaking, as the date could be overwritten over time. I have a test that checks the date, where I use regex to verify if the field itself could be a date. This is just a suggestion.

Comment (subhashkhileri) [reply e2e-tests/playwright/e2e/plugins/acr.spec.ts]: updated

**Review by josephca — APPROVED** [review]
/lgtm
/approve

File: e2e-tests/playwright/utils/UIhelper.ts [hunk 2]
```diff
@@ -160,20 +160,17 @@ export class UIhelper {
...
+    cellTexts: string[] | RegExp[],
   ) {
-    const uniqueCell = this.page
-      .locator(UIhelperPO.MuiTableCell)
-      .locator(`text=${uniqueRowText}`);
-    await uniqueCell.scrollIntoViewIfNeeded();
-    const row = uniqueCell.locator('xpath=ancestor::tr');
-
+    const row = this.page.locator(
+      `xpath=//tr[child::td[text()='${uniqueRowText}']]`,
```
Comment (rnapoles-rh) [inline e2e-tests/playwright/utils/UIhelper.ts:167]: We should avoid the use of xpaths, if the UI changes the test breaks. If possible lets us the object ID instead.

**Review by rnapoles-rh — CHANGES_REQUESTED** [review]

File: e2e-tests/playwright/utils/UIhelper.ts (hunk 2 above)
Comment (subhashkhileri) [reply e2e-tests/playwright/utils/UIhelper.ts]: @rnapoles-rh Locator updated.

**Review by rnapoles-rh — APPROVED** [review]
/lgtm

**Pull request merged**